### PR TITLE
bpo-26205: Specify the number of nested scopes

### PR DESCRIPTION
**Pull request by nanjekyejoannah**
https://bugs.python.org/issue26205

### CODE REVIEW

File: Doc/tutorial/classes.rst [hunk 1]
```diff
@@ -114,8 +114,8 @@ accessible.  "Directly accessible" here means that an unqualified reference to a
 name attempts to find the name in the namespace.
 
 Although scopes are determined statically, they are used dynamically. At any
-time during execution, there are at least three nested scopes whose namespaces
-are directly accessible:
+time during execution, At any time during execution, there are various nested
```
Comment (terryjreedy) [inline Doc/tutorial/classes.rst]:
```suggestion
time during execution, At any time during execution, there are 3 or 4 nested
```
The list that follows has 4, 1 of which is only conditionally present, making 3 or 4.  I said more on the issue.

**Review by terryjreedy — CHANGES_REQUESTED** [review]
I approve with the requested change.

**Review by terryjreedy — APPROVED** [review]

**Comment by terryjreedy** [comment]
Please merge with a changed title, as I discussed on #21340.

**Comment by nanjekyejoannah** [comment]
Thanks for the review @terryjreedy

**Pull request merged**